setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.129.3
```

Comment:
we will have to double bump before next release

Reply:
I just added the commit of bump to restore 2.129.3 on the branch